pkg/install/daemonset.go

```diff
@@ -68,10 +68,9 @@ func DaemonSet(namespace string, opts ...podTemplateOption) *appsv1.DaemonSet {
 			},
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
-					Labels: map[string]string{
-						"name":      "restic",
-						"component": "velero",
-					},
+					Labels: podLabels(c.labels, map[string]string{
+						"name": "restic",
+					}),
```

Comment:
Any reason to remove the label `component=velero`?

Reply:
I'm not removing it, I just aggregated the labels usage. 
The function `Labels` called in `podLabels` adds this label by default.
I'm just adding the label that wasn't a default here.